flink-connectors/flink-hbase/src/main/java/org/apache/flink/addons/hbase/HBaseTableSchema.java

```diff
@@ -54,7 +54,7 @@
 	 * @param qualifier the qualifier name
 	 * @param clazz     the data type of the qualifier
 	 */
-	void addColumn(String family, String qualifier, Class<?> clazz) {
+	public void addColumn(String family, String qualifier, Class<?> clazz) {
```

Comment:
The reason is we want to expose `HBaseTableSchema` to users. Because `HBaseUpsertSinkFunction` accepts `HBaseTableSchema` as a parameter. If we want users to construct a `HBaseUpsertSinkFunction` and used in DataStream,  core methods of `HBaseTableSchema` should also be exposed. 
What do you think?

Reply:
I see, makes sense.